openedx/features/content_tagging/rules.py

```diff
@@ -35,27 +36,108 @@ def is_taxonomy_admin(user: User, taxonomy: oel_tagging.Taxonomy = None) -> bool
     return False
 
 
+def is_global_taxonomy_admin(user: User) -> bool:
+    """
+    Returns True if the given user is a Global Taxonomy Admin.
+
+    Global Taxonomy Admins include global staff and superusers.
+    """
+    if oel_tagging.is_taxonomy_admin(user):
+        return True
```

Comment:
This method returns `None` implicitly if the user is not a taxonomy admin.. could you write it like this instead?
```suggestion
    return oel_tagging.is_taxonomy_admin(user)
```

Reply:
Done 067c5bee849de9c57454387e66c14e20b054ddd7